cmake/externalFunc.cmake

```
@@ -26,11 +26,17 @@ macro(addExternalFolder NAME FOLDER )
     # Check if install prefix has changed. If yes, force installing the externals again
     if( DEFINED CACHED_INSTALL_PREFIX )
         if( NOT "${CACHED_INSTALL_PREFIX}" STREQUAL "${CMAKE_INSTALL_PREFIX}" )
-            message(STATUS "[addExternalFolder] CMAKE_INSTALL_PREFIX has changed (from ${CACHED_INSTALL_PREFIX} to ${CMAKE_INSTALL_PREFIX}), reinstalling dependency")
+            message(STATUS "[addExternalFolder] CMAKE_INSTALL_PREFIX has changed (from ${CACHED_INSTALL_PREFIX} to ${CMAKE_INSTALL_PREFIX}), reinstalling dependencies")
             set( UPDATE_EXTERNAL ON)
         endif()
     endif()
 
+    # Check if install directory is empty. If yes, force installing the externals again
```

This comment is not representative of what is done. Here, you search if one "external" (${NAME}) is installed.
```suggestion
    # Check if the external ${NAME} is installed. If not, force installing the externals again
```